ietf/templates/group/meetings.html

```diff
@@ -85,6 +85,7 @@ <h2 class="mt-3" id="pastmeets">Past Meetings</h2>
         </table>
     {% endif %}
     <p class="alert alert-info my-3">
-        This page shows meetings within the last four years. For earlier meetings, please see the proceedings.
+        This page shows meetings within the last four years. For earlier meetings, please see the
+        <a href="https://www.ietf.org/how/meetings/past/">proceedings</a>.
```

Comment:
You could use `settings.IETF_HOST_URL` instead of hardcoding, but we're not very consistent with it.

Reply:
I looked at the pulldown for "Meeting proceedings", and it is hard coded there, so I did the same.

Reply:
we should be more consistent with it, but lets get that with a concentrated pass through the code.